### PR TITLE
Changes made to the Legend Doku Text

### PR DESCRIPTION
**Pull request by momotosushi**
Deleted 2 Images, by changing their path to 2 existing images, that looked the same and we already had.

In the "Legend in the toolbar" part, the button config screenshot was deleted, because it is explained in detail( in the referred "Button" Element), how to do so.

Maybe discuss this in person, in our meeting, later!

### CODE REVIEW

File: de/elements/basic/legend.rst [hunk 1]
```diff
@@ -58,23 +58,16 @@ Die Legende kann auch als Button in der Werkzeugleiste eingebunden werden. Hierf
...
-     :scale: 80
-
 Mit diesen Einstellungen sieht das Ergebnis in der Anwendung wie folgt aus:
 
-.. image:: ../../../figures/de/legend_example_toolbar.png
+.. image:: ../../../figures/de/legend.png
      :scale: 80
 
-In der Oberen Werkzeugleiste (Toolbar) ist der Button für das Legenden-Element zu sehen. Sobald auf den Button geklickt wird, öffnet sich ein Dialog mit der Legende.
+In der Oberen Werkzeugleiste (Toolbar) ist der Button für das Legenden-Element zu sehen. Sobald auf den Button geklickt wird, öffnet sich ein Dialog mit der Legende(wird in unserem Beispiel automatisch geöffnet).
```
Comment (astroidex) [inline de/elements/basic/legend.rst]: ...Dialog mit der Legende. Durch die Einstellung "Automatisch öffnen" wird der Dialog beim Start der Anwendung direkt angezeigt.

File: en/elements/basic/legend.rst [hunk 2]
```diff
@@ -59,28 +59,21 @@ The legend element can be integrated with a button in the toolbar. First step: O
...
-Following the above instructions, the result in the application looks like this:
+As soon as this element has been implemented into the map area, a :ref:`button` has to be implemented into the toolbar section.
+Following these instructions, the result in the application looks like this:
 
-.. image:: ../../../figures/legend_example_toolbar.png
+.. image:: ../../../figures/legend.png
      :scale: 80
 
-The toolbar shows the button for the legend element. If the button is clicked, the dialog with the generated legend opens.
+The toolbar shows the button for the legend element. If the button is clicked, the dialog with the generated legend opens(will open automatically in our example).
```
Comment (astroidex) [inline en/elements/basic/legend.rst]: With the "Open automatically" setting, the dialog is displayed directly when the application is started.

**Review by astroidex — CHANGES_REQUESTED** [review]

**Comment by astroidex** [comment]
There is one heading "Legend in the toolbar"
- maybe it is missleading. As the dialogs opens in the map area?
- maybe better "Legend via button in toolbar"
- or is it too much and we delete the section @rschwammborn ?
https://doc.mapbender.org/en/elements/basic/legend.html#legend-in-the-toolbar

**Comment by rschwammborn** [comment]
Good remark, please change the heading accordingly.

Alternatively (or in the future), we could delete the button section. I'm against this, as deleting only one example leads to incomplete information about configuration options. Deleting the whole section about the configuration examples would be a bit too much. Maybe we should add a "default example" page similar to the button page for the sidepane. Closest we have right now is [this section](https://doc.mapbender.org/en/backend/applications/layouts.html#configuration-of-the-sidepane). Maybe we could add an example section below about how to include an element into the sidepane and use crosslinks in this (and similar) case(s) to it.

**Pull request merged**